tests/sample_data.py

```diff
@@ -0,0 +1,211 @@
+from salmon_occurrence import salmon_db, Region, Taxon, ConservationUnit, Population
+import re
+import json
+
+
+# square test regions.
+# watersheds 1 and two overlap and are inside basin 1.
+# watershed 3 is off by itself
+
+WAT1 = {
+    "kind": "watershed",
+    "name": "Watershed 1",
+    "code": "WAT1",
+    "boundary": "POLYGON((0 0, 0 10, 10 10, 10 0, 0 0))",
+    "outlet": "POINT(0 0)",
+}
+
+WAT2 = {
+    "kind": "watershed",
+    "name": "Watershed 2",
+    "code": "WAT2",
+    "boundary": "POLYGON((5 5, 5 15, 15 15, 15 5, 5 5))",
+    "outlet": "POINT(5 5)",
+}
+
+WAT3 = {
+    "kind": "watershed",
+    "name": "Watershed 3",
+    "code": "WAT3",
+    "boundary": "POLYGON((20 20, 20 30, 30 30, 30 20, 20 20))",
+    "outlet": "POINT(20 20)",
+}
+
+BAS1 = {
+    "kind": "basin",
+    "name": "Basin 1",
+    "code": "BAS1",
+    "boundary": "POLYGON((0 0, 0 15, 15 15, 15 0, 0 0))",
+    "outlet": "POINT(0 0)",
+}
+
+
+# make a sample region into an ORM Region for database insertion
+def make_region(params):
```

Comment:
We could simplify this function by listing content of `params` as keyword args, and passing them in with the spread operator (`**`), e.g., `make_region(**WAT1)`. This also makes it easy to pass in values for testing.